### PR TITLE
Update all sample codes to add await and async for initialize function

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6692643322

image

File: samples/app-complete-auth/csharp/AppCompleteAuth/Pages/config.cshtml [hunk 1]
```diff
@@ -10,18 +10,18 @@
...
     <script>
-        microsoftTeams.app.initialize();
+        microsoftTeams.app.initialize().then(() => {
+            function ssoLogin(){
+                microsoftTeams.authentication.notifySuccess('sso');
+            }
+            function facebookLogin() {
+                microsoftTeams.authentication.notifySuccess('facebooklogin');
+            }
+            function usingCredentialsLogin() {
```
Comment (Prithvi-MSFT) [inline samples/app-complete-auth/csharp/AppCompleteAuth/Pages/config.cshtml:21]: remove extra line

File: samples/app-complete-auth/nodejs/views/fb-auth.ejs [hunk 2]
```diff
@@ -8,7 +8,8 @@
         integrity="sha384-lIk8T3uMxKqXQVVfFbiw0K/Nq+kt1P3NtGt/pNexiDby2rKU6xnDY8p16gIwKqgI" ,
         crossorigin="anonymous"></script>
     <script>
-        microsoftTeams.app.initialize();
+        microsoftTeams.app.initialize().then(() => {
```
Comment (Prithvi-MSFT) [inline samples/app-complete-auth/nodejs/views/fb-auth.ejs:11]: please try once with await microsoftTeams.app.initialize()

File: samples/app-complete-sample/csharp/wwwroot/Page/composeExtensionSettings.html [hunk 3]
```diff
@@ -14,7 +14,8 @@
     <button onclick="selectHeroCards()">Hero Cards</button>
     <script>
         $(document).ready(function () {
-            microsoftTeams.app.initialize();
+            microsoftTeams.app.initialize().then(() => {
```
Comment (Prithvi-MSFT) [inline samples/app-complete-sample/csharp/wwwroot/Page/composeExtensionSettings.html:17]: please try once with await microsoftTeams.app.initialize()

File: samples/app-complete-sample/csharp/wwwroot/Page/popUpSignin.html [hunk 4]
```diff
@@ -18,7 +18,8 @@
     <script src="https://res.cdn.office.net/teams-js/2.0.0/js/MicrosoftTeams.min.js" crossorigin="anonymous"></script>
 
     <script>
-        microsoftTeams.app.initialize();
+        microsoftTeams.app.initialize().then(() => {
```
Comment (Prithvi-MSFT) [inline samples/app-complete-sample/csharp/wwwroot/Page/popUpSignin.html:21]: please try once with await microsoftTeams.app.initialize()

File: samples/graph-app-installation-lifecycle/csharp/AppInstallation/Views/Home/Index.cshtml [hunk 5]
```diff
@@ -86,7 +86,8 @@
 
 <script type="text/javascript">
     $(document).ready(function () {
-        microsoftTeams.app.initialize();
+        microsoftTeams.app.initialize().then(() => {
```
Comment (Prithvi-MSFT) [inline samples/graph-app-installation-lifecycle/csharp/AppInstallation/Views/Home/Index.cshtml:89]: please try once with await microsoftTeams.app.initialize() and make the function async

File: samples/graph-chat-lifecycle/csharp/ChatLifecycle/Views/Home/Index.cshtml [hunk 6]
```diff
@@ -30,7 +30,8 @@
 
 <script>
   
-    microsoftTeams.app.initialize();
+    microsoftTeams.app.initialize().then(() => {
```
Comment (Prithvi-MSFT) [inline samples/graph-chat-lifecycle/csharp/ChatLifecycle/Views/Home/Index.cshtml:33]: please try once with await microsoftTeams.app.initialize()

**Review by Prithvi-MSFT — CHANGES_REQUESTED** [review]

File: samples/graph-chat-lifecycle/csharp/ChatLifecycle/Views/Home/Index.cshtml (hunk 6 above)
Comment (Prithvi-MSFT) [reply samples/graph-chat-lifecycle/csharp/ChatLifecycle/Views/Home/Index.cshtml:33]: Please update at all places if above code works

Comment (Harikrishnan-MSFT) [reply samples/graph-chat-lifecycle/csharp/ChatLifecycle/Views/Home/Index.cshtml:33]: Hi @Prithvi-MSFT 

I have tried, as per your PR comments, to **await microsoftTeams.app.initialize()**, but i am getting this error.


image

image

File: samples/app-complete-auth/csharp/AppCompleteAuth/Pages/config.cshtml (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/app-complete-auth/csharp/AppCompleteAuth/Pages/config.cshtml:21]: Fixed

**Review by Prithvi-MSFT — APPROVED** [review]

**Review by Pawank-MSFT — APPROVED** [review]
looks correct., Approving!

**Comment by Pawank-MSFT** [comment]
Executed Build pipeline
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6733625246

**Comment by Harikrishnan-MSFT** [comment]
Executed Build pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6741388547
image

**Review by Wajeed-msft — APPROVED** [review]

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6939481049

**Comment by Harikrishnan-MSFT** [comment]
Bug Link - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/issues/988

**Pull request merged**